README.md

```diff
@@ -223,9 +235,9 @@ vous trouverez sur cette plateform des cours et examens qui vous accompagnerons
     * [Ali Hamdi](https://www.youtube.com/channel/UChiajEj7cSVoY0k2FDlM2zA) Web developement & web design tutorials
     * [Hassouna academy](https://www.youtube.com/user/HassounaAcademy/playlists): Everything related to computer and its sciences, web design and software
     * [Programming with Mosh](https://www.youtube.com/user/programmingwithmosh) programming tutorials
-    
-    
-    
+
```

Comment:
Why the unnecessary line jumps ? 😄

Reply:
Seems like i forgot to tidy up the rows XD. fixed